MANIFESTO.md

```diff
@@ -0,0 +1,38 @@
+# Manifesto
+As Hitta.se's development team, we believe in...
+
+1. **Testing matters**
+   - We exponentially increase the amount of rework when we try to find and fix bugs later in any lifecycle.
+   - That's why we test early and test often.
+   - We always dedicate time to expand our automated test suites by building both unit and end-to-end tests.
+2. **Quality above all**
+   - Quality takes priority over features and time.
+   - Quality has many facets, such as security, performance and even a great user experience.
+   - Code we write in the morning may be in production by the afternoon.
+3. **No islands**
+   - We collaborate on design.
+   - We make sure knowledge is shared.
+   - We do continuous code reviews.
+   - We commit and push frequently.
+   - We give honest feedback.
+   - Processes are a means to help individuals and interactions.
+   - Learn from others and help others learn from you.
+4. **Coding is an art**
+   - Designing solutions for the right scale, don't over-architect.
+   - Aim for simplicity, honor KISS principle.
+   - Design patterns are inspirations, not solutions.
```

Comment:
- Aggregate data at or close to the source - transfer as little as possible

(I.e. - aggregate *in* MongoDB, not in a service using MongoDB)

Reply:
Added!